internal/app/server/open_saves_test.go

```diff
@@ -124,6 +126,8 @@ func TestOpenSaves_HealthCheck(t *testing.T) {
 	if want := healthgrpc.HealthCheckResponse_NOT_SERVING; got.Status != want {
 		t.Errorf("hc.Check got: %v, want: %v", got.Status, want)
 	}
+	// Sleep long enough to free up the connection
+	time.Sleep(serviceConfig.ShutdownGracePeriod)
```

Comment:
Perhaps we don't need to sleep in tests?

Reply:
We need to sleep if we set the grace period to be 1s. If we set it to 0s, then we would not need to sleep.